src/game/models/vector.rs

```diff
@@ -0,0 +1,144 @@
+use rand;
```

Comment:
Let's use the standard grouping for imports: https://github.com/rust-lang-nursery/fmt-rfcs/issues/24#issuecomment-252161813

Three sections, with one blank line after each:

- Standard library imports
- External crate imports
- Internal imports

So in this instance, `rand` would be imported after the `std::`'s and alphabetically after `piston_window` in the "3rd party" group.